test/new-e2e/tests/agent-platform/common/file-manager/client.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package filemanager implement interfaces to run install-script tests
+package filemanager
+
+import (
+	"io/fs"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+)
+
+// Client implement filemanager interface for VMs
+type Client struct {
+	host *components.RemoteHost
+}
+
+// NewClientFileManager create a new file manager using the client
+// Note: The file operations will be restricted to the permissions of the client user
+func NewClientFileManager(host *components.RemoteHost) *Client {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
We know it is a `filemanager` by its package
```suggestion
func NewClient(host *components.RemoteHost) *Client {
```